test/CardRange.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Created by bjorn on 07.12.20.
+//
+
+#include <gtest/gtest.h>
+#include "src/gamelogic/gamemodel/CardRange.hpp"
+
+TEST(card_range_test, select_policy) {
+    GameModel::Board board{};
+
+    GameModel::CardRange cardRange{board, 1};
+    EXPECT_TRUE(cardRange.selectForPolicy(GameModel::CardType::Fascist));
+    EXPECT_FALSE(cardRange.selectForPolicy(GameModel::CardType::Liberal));
+}
```

Comment:
Der Fall, bei dem eine nicht vorhandene Karte ausgewählt wurde ist nicht dabei. Da könnte man noch einen Test schreiben

Reply:
Ist der nicht abgededeckt, da ich probiere eine Karte zu selektieren, die es nicht in der Cardrange gibt?